docs/api/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "example",
   "title": "Example Integration",
   "version": "1.0.0",
+  "readme": "/docs/README.md",
```

Comment:
@neptunian Would that work as an API endpoint? This entry exists, if there is a readme. If there is no README.md file, this entry does not exist and we should fall back to the description.

I wonder if we should call this entry differently like `docs_index` or similar?

Reply:
Personally, I like `readme` for this. We've currently got:

`name`: essentially an id (e.g. `postgres`)
`title`: Short stylized string representation of the package (e.g. `PostgreSQL`)
`description`: medium(? not sure how to quantify) length text. Used in our cards for results

We need something for "Long description (pages in length is fine) that allows stylized markup (specifically Markdown)". I think `readme` works for this.

Authors might reasonably expect `docs` or `documentation` to point to their docs site or a page which uses a layout like this image which is common. We wouldn't want something like that on the details page, but it's very common for projects to use this style in documentation.

<img width="718" alt="Screen Shot 2019-10-22 at 5 10 42 PM" src="https://user-images.githubusercontent.com/57655/67333628-e3a2c580-f4ee-11e9-8146-27d4fb0bf78b.png">


Looking at the [PostgreSQL 12 docs](https://www.postgresql.org/docs/12/index.html) I don't think we'd want something like this in the details view

<img width="696" alt="Screen Shot 2019-10-22 at 5 15 31 PM" src="https://user-images.githubusercontent.com/57655/67334044-907d4280-f4ef-11e9-969c-0e5bccb6433a.png">

The [PostgreSQL README](https://github.com/postgres/postgres/blob/master/README) isn't very attractive either, but I think it's more useful and appropriate for the details page.

Begin README paste:
-----------------

PostgreSQL Database Management System
=====================================

This directory contains the source code distribution of the PostgreSQL
database management system.

PostgreSQL is an advanced object-relational database management system
that supports an extended subset of the SQL standard, including
transactions, foreign keys, subqueries, triggers, user-defined types
and functions.  This distribution also contains C language bindings.

PostgreSQL has many language interfaces, many of which are listed here:

	https://www.postgresql.org/download

See the file INSTALL for instructions on how to build and install
PostgreSQL.  That file also lists supported operating systems and
hardware platforms and contains information regarding any other
software packages that are required to build or run the PostgreSQL
system.  Copyright and license information can be found in the
file COPYRIGHT.  A comprehensive documentation set is included in this
distribution; it can be read as described in the installation
instructions.

The latest version of this software may be obtained at
https://www.postgresql.org/download/.  For more information look at our
web site located at https://www.postgresql.org/.

-----------------
End README paste

I realize we not using these repo files specifically, but I think it points to a convention of what people expect in a README and that aligns well with what we're putting in that section of the details view.

There's also precedent in [Rust's `Cargo.toml` manifest](https://doc.rust-lang.org/cargo/reference/manifest.html) format. From their [`[package]` section](https://doc.rust-lang.org/cargo/reference/manifest.html#package-metadata)

```
# A short blurb about the package. This is not rendered in any format when
# uploaded to crates.io (aka this is not markdown).
description = "..."

# These URLs point to more information about the package. These are
# intended to be webviews of the relevant data, not necessarily compatible
# with VCS tools and the like.
documentation = "..."
homepage = "..."
repository = "..."

# This points to a file under the package root (relative to this `Cargo.toml`).
# The contents of this file are stored and indexed in the registry.
# crates.io will render this file and place the result on the crate's page.
readme = "..."
```